tests/functions/module_statistical.test.ts

```diff
@@ -1847,6 +1847,142 @@ describe.each([["QUARTILE"], ["QUARTILE.INC"], ["QUARTILE.EXC"]])("%s formula",
   });
 });
 
+describe("RANK formula", () => {
+  test("functional tests on simple arguments with column data", () => {
+    const grid = {
```

Comment:
would be best to format this grid like others (Column A and B next to each other)